system-tests/test/deposit-redemption.test.ts

```diff
@@ -167,11 +204,16 @@ describe("System Test - Deposit and redemption", () => {
       // Since the reveal deposit logic does not perform SPV proof, we
       // can reveal the deposit transaction immediately without waiting
       // for confirmations.
-      await TBTC.revealDeposit(
-        depositUtxo,
-        deposit,
-        electrumClient,
-        depositorBridgeHandle
+      const rawDepositTransaction = await electrumClient.getRawTransaction(
+        depositUtxo.transactionHash
+      )
+      const depositRawTxVectors = extractBitcoinRawTxVectors(
+        rawDepositTransaction
+      )
+      await depositorBridgeHandle.revealDeposit(
```

Comment:
All deposit creation should go in the new fashion: https://docs.threshold.network/app-development/tbtc-v2/tbtc-sdk/guides/deposit-and-mint. The only problematic part may be revealing without using the `TBTCVault` but this can be worked around using `sdk.tbtcContracts.bridge.revealDeposit`.

Reply:
Done in 5691a18c4dff48f0fcfc198c32e245ed28903472.